website/docs/user-group-role/access-control/permissions.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "About permissions"
+description: "Learn about global and object permissions in authentik."
+---
+
+Permissions are the central components in all access control systems, the lowest-level component, the controlling piece of access data. Permissions are assigned to (or removed from!) to define exactly WHO can do WHAT to WHICH part of the overall software system.
+
+## Fundamentals of authentik permissions
+
+There are two main types of permissions in authentik:
+
+-   [**Global permissions**](#global-permissions)
+-   [**Object permissions**](#object-permissions)
+
+### Global permissions
+
+Global permissions define who can do what on a global level, across the entire system. Some examples in authentik are the ability to add new [flows](../../flow/index.md) or to create a URL for users to recover their login credentials.
```

Comment:
Suggestion: Remove the comma here and see if it doesn't flow better.  Or remove the repetition of "global" entirely to avoid defining a concept using the very words with which you label it.

"Global permissions define who can do what across the entire system."